templates/resources/includes/deploy-task-engine-run-script.j2

```diff
@@ -100,10 +100,35 @@ else
         echo server.hostname=${HOSTNAME}${HOSTNAME_SUFFIX}
       } >> ${APP_HOME}/conf/{{ boot_conf }}
     fi
-    echo "Updating server port property"
+    if [[ "${USE_EXTERNAL_CENTRAL_CONFIG}" != "" ]]; then
+      echo "Updating USE_EXTERNAL_CENTRAL_CONFIG property"
+      sed -i "s#xl.spring.cloud.external-config\=.*##" ${APP_HOME}/conf/{{ boot_conf }}
+      {
+        echo xl.spring.cloud.external-config=${USE_EXTERNAL_CENTRAL_CONFIG}
+      } >> ${APP_HOME}/conf/{{ boot_conf }}
+    fi
+    if [[ "${CENTRAL_CONFIG_URL}" != "" ]]; then
+      echo "Updating CENTRAL_CONFIG_URL property"
+      sed -i "s#xl.spring.cloud.uri\=.*##" ${APP_HOME}/conf/{{ boot_conf }}
+      {
+        echo xl.spring.cloud.uri=${CENTRAL_CONFIG_URL}
+      } >> ${APP_HOME}/conf/{{ boot_conf }}
+    fi
+    if [[ "${CENTRAL_CONFIG_ENCRYPT_KEY}" != "" ]]; then
+      if ! grep -q xl.spring.cloud.encrypt.key ${APP_HOME}/conf/{{ boot_conf }}; then
+        echo "Updating CENTRAL_CONFIG_ENCRYPT_KEY property"
+        sed -i "s#xl.spring.cloud.encrypt.key\=.*##" ${APP_HOME}/conf/{{ boot_conf }}
```

Comment:
I think this is not required since we check if the property is not there

Reply:
correct and removed the addition filtering